index.html

```diff
@@ -176,40 +171,43 @@ <h2>LET’S CONNECT</h2>
                 </ul>
                 <a id="my_resume" href="#">Get my resume</a>
               </div>
-              <div id="my_skills">
-                  <div id="languages_skills">
-                      <div id="spec_name">
-                          <p>Languages</p>
-                          <div><img src="statics/images/fleche_dessous.png" alt="selected"></div>
-                      </div>
-                      <ul id="programmes">
-                        <li class="pro">
-                            <img src="statics/images/java.png" alt="javascript image">
-                            <p>JavaSript</p>
-                        </li>
-                        <li class="pro">
-                            <img src="statics/images/html.png" alt="html image">
-                            <p>HTML</p>
-                        </li>
-                        <li class="pro">
-                            <img src="statics/images/css.png" alt="css image">
-                            <p>CSS</p>
-                        </li>
-                      </ul>
-                  </div>
-                  <div id="frameworks">
-                      <p>Frameworks</p>
-                      <div><img src="statics/images/select.png" alt="select"></div>
-                  </div>
-                  <div id="l1"> <hr/></div>
-                  <div id="sk">
-                    <p>Skills</p>
-                    <div><img src="statics/images/select.png" alt="select"></div>
-                  </div>
-                  <div class="ligne" id="l2"> <hr/></div>
+            </div>
+            <div id="my_skills">
+              <div id="languages_skills">
+                <div id="spec_name">
+                    <p>Languages</p>
+                    <div><img src="statics/images/fleche_dessous.png" alt="selected"></div>
                 </div>
-            </section>
-            <section class="form-section">
+                <ul id="programmes">
+                     <li class="pro">
+                       <img src="statics/images/java.png" alt="javascript image">
+                       <p>JavaSript</p>
+                     </li>
+                     <li class="pro">
+                       <img src="statics/images/html.png" alt="html image">
+                       <p>HTML</p>
+                     </li>
+                     <li class="pro">
+                       <img src="statics/images/css.png" alt="css image">
+                       <p>CSS</p>
+                     </li>
+                </ul>
```

Comment:
- [ ] The languages section looks quite different than the original design the alignment change to row and the containers turn to squares with the logo at the top-left corner.
![Captura de pantalla_2021-09-24_21-47-40](https://user-images.githubusercontent.com/53111464/134755381-31b8057b-031e-4ae3-9102-9413b5630135.png)

**Figma**
![Captura de pantalla_2021-09-24_21-47-23](https://user-images.githubusercontent.com/53111464/134755394-098bdaf1-270b-47b4-9944-76f621329e48.png)

Reply:
- [OPTIONAL] The container also has a rounded corner.
![Captura de pantalla_2021-09-24_21-49-36](https://user-images.githubusercontent.com/53111464/134755441-1b44dde0-0d62-4228-bd5b-313d88bcdcd4.png)